driver/driver_test.go

```diff
@@ -0,0 +1,64 @@
+package driver
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/brimsec/zq/scanner"
+	"github.com/brimsec/zq/zbuf"
+	"github.com/brimsec/zq/zio/zngio"
+	"github.com/brimsec/zq/zng"
+	"github.com/brimsec/zq/zng/resolver"
+	"github.com/brimsec/zq/zql"
+	"github.com/stretchr/testify/assert"
+)
+
+type counter struct {
+	n int
+}
+
+func (c *counter) Write(*zng.Record) error {
+	c.n++
+	return nil
+}
+
+func TestMuxDriver(t *testing.T) {
+	input := `#0:record[_path:string,ts:time]
```

Comment:
nit: would you start the string literal on the next line so it reads a little easier down the page?:
```
  input := `
#0:record ...
0:...
```